server/notifications/user_notifier.go

```diff
@@ -51,9 +60,16 @@ func (n *UsersNotifier) Notify(ctx context.Context) {
 		"notifying users by email",
 		slog.Any("userCount", len(users)),
 	)
+
+	n.messageFormatter.SetRate(rate)
 	for _, user := range users {
-		message := NewRateMessage(rate)
-		if err := n.mailClient.SendEmail(ctx, user.Email, message.String()); err != nil {
+		err := n.mailClient.SendEmail(
```

Comment:
please, add support for multiple receivers as described [here](https://github.com/GenesisEducationKyiv/software-engineering-school-4-0-Hukyl/pull/5#discussion_r1649594755) or add support for concurrent message sending